server/chat-commands.js

```diff
@@ -473,14 +473,15 @@ const commands = {
 	w: 'msg',
 	msg(target, room, user, connection) {
 		if (!target) return this.parse('/help msg');
+		if (!target.includes(',')) {
+			this.errorReply("You forgot the comma.");
+			return this.parse('/help msg');
+		}
 		target = this.splitTarget(target);
 		let targetUser = this.targetUser;
 		if (this.targetUsername === '~') {
 			this.room = Rooms.global;
 			this.pmTarget = null;
-		} else if (!target) {
```

Comment:
This check still does need to be done, just with a different message, like "You can't leave your message empty."

Reply:
Isn't that handled here? https://github.com/Zarel/Pokemon-Showdown/blob/d6b4705870849d54543038321d3b5c898beea0b6/server/chat.js#L1162-L1165

Reply:
Good point. Yeah, that should be good.